frontend/app/module/election/page/ElectionStatusPage.test.tsx

```diff
@@ -20,17 +20,84 @@ describe("ElectionStatusPage", () => {
     overrideOnce("get", "/api/elections/1", 200, electionDetailsMockResponse);
   });
 
-  test("Finish input not visible when not finished", async () => {
+  test("Renders page correctly and does not show finish input", async () => {
     renderElectionStatusPage();
 
+    overrideOnce("get", "/api/elections/1/status", 200, {
+      statuses: [
+        {
+          id: 1,
+          status: "not_started",
+        },
+        {
+          id: 2,
+          status: "not_started",
+        },
+        {
+          id: 3,
+          status: "first_entry_unfinished",
+        },
+        {
+          id: 4,
+          status: "first_entry_in_progress",
+        },
+      ],
+    });
+
     // Wait for the page to be loaded
     expect(await screen.findByRole("heading", { level: 1, name: "Eerste zitting" }));
 
-    // Test that the message doesn't exist
+    const items = [...screen.getByTestId("shortcuts").children];
+    expect(items[0]).toHaveTextContent("Snelkoppelingen");
+    expect(items[1]).toHaveTextContent("Niet afgeronde invoer (1)");
+    expect(items[2]).toHaveTextContent("Invoer bezig (1)");
+    expect(items[3]).toHaveTextContent("Eerste invoer klaar (0)");
+    expect(items[4]).toHaveTextContent("Werkvoorraad (2)");
+
+    expect(screen.getByTestId("progressbar-all")).toBeInTheDocument();
+    const bars = [...screen.getByTestId("multi-outer-bar").children];
+    const expectedData = [
+      { percentage: 0, class: "definitive" },
+      { percentage: 25, class: "in-progress" },
+      { percentage: 25, class: "unfinished" },
+      { percentage: 50, class: "not-started" },
+    ];
+    bars.forEach((bar, index) => {
+      expect(bar.getAttribute("style")).toEqual(`width: ${expectedData[index]?.percentage}%;`);
+      expect(bar.classList.contains(`${expectedData[index]?.class}`)).toBeTruthy();
+    });
+
+    const tables = [...screen.getByRole("article").children];
+    expect(tables[0]).toHaveTextContent("Niet afgeronde invoer (1)");
+    expect(tables[0]).toHaveTextContent(/Nummer/);
+    expect(tables[0]).toHaveTextContent(/Stembureau/);
+    expect(tables[0]).toHaveTextContent(/Invoerder/);
+    expect(tables[0]).toHaveTextContent(/35/);
+    expect(tables[0]).toHaveTextContent(/Testschool/);
+    expect(tables[0]).toHaveTextContent(/1e invoer/);
```

Comment:
Updated in b3933ef16fe57784c0178b1981b7f3f12399b2b1

Reply:
My comment was too vague, so the change isn't entirely what I had in mind. And there was some more figuring out to do, because it's not straightforward to select a specific page + heading.

So instead of trying to do a better job describing, I made a commit: 4aea9bc8cd2b01b4af280c5c2a9fe62dd8755372